spec/lib/science_wire_harvester_spec.rb

```diff
@@ -1,8 +1,7 @@
-
 describe ScienceWireHarvester, :vcr do
   let(:author_without_seed_data) { create :author, emails_for_harvest: '' }
   let(:author_with_seed_email) { create :author }
-  let(:author) { create :author }
+  let(:author) { create :author, cap_import_enabled: false }
```

Comment:
I can understand this change is intended to be a no-op after changing the default in the factory.  Just curious about why this cannot use the default in the factory, if you care to look into it?

Reply:
Just because the test was specifically asking about `cap_import_enabled`.

Reply:
I suppose the next question is whether or not the factory should have the same default value as the db-schema.  It looks like `db/schema.rb` sets no default and it allows `NULL`; in effect, I _think_ the default is a `falsey` or `nil` value.  Perhaps the factory should not explicitly set it?
```ruby
> Author.new.cap_import_enabled
=> nil
```

Reply:
Seems like the value might be set in `Author.fetch_from_cap_and_create` via the cap authors poller, but it depends entirely on the data in the CAP-API retrieved.